Documentation/hacking.md

```diff
@@ -52,20 +52,22 @@ Alternatively, you can build rkt in a Docker container with the following comman
 Replace $SRC with the absolute path to your rkt source code:
 
 ```
-$ sudo docker run -v $SRC:/opt/rkt -i -t golang:1.4 /bin/bash -c "apt-get update && apt-get install -y coreutils cpio squashfs-tools realpath && cd /opt/rkt && go get github.com/appc/spec/... && ./build"
+$ sudo docker run -v $SRC:/opt/rkt -i -t golang:1.4 /bin/bash -c "apt-get update && apt-get install -y coreutils cpio squashfs-tools realpath autoconf libcapture-tiny-perl file && cd /opt/rkt && go get github.com/appc/spec/... && ./autogen.sh && ./configure && make"
 ```
 
 ### Building systemd in stage1 from the sources
 
-By default, rkt gets systemd from a CoreOS image to generate stage1. But it's also possible to build systemd from the sources. To do that, you can set the following environment variables:
+By default, rkt gets systemd from a CoreOS image to generate stage1. But it's also possible to build systemd from the sources.
+After running `./autogen.sh` you can select the following options:
 
-* `RKT_STAGE1_USR_FROM=none|coreos|src`: choose how to generate systemd for stage1
-* `RKT_STAGE1_SYSTEMD_VER=v219|v220|master`: if `RKT_STAGE1_USR_FROM=src`, choose the systemd branch or tag to build
+* ./configure --with-stage1=none|coreos|src|host`: choose how to generate systemd for stage1 (default: 'coreos')
+* ./configure --with-stage1-systemd-version=version`: if stage1 is set to build from src, choose the systemd branch or tag to build (default: 'v220')
+* ./configure --with-stage1-systemd-src=git-path`: if stage1 is set to build from src, systemd git repository's address (default: 'https://github.com/systemd/systemd.git')
 
 Example:
 
 ```
-RKT_STAGE1_USR_FROM=src RKT_STAGE1_SYSTEMD_VER=v220 ./build
+./autogen.sh && ./configure --with-stage1=src --with-stage1-systemd-version=HEAD --with-stage1-systemd-src=$HOME/src/systemd && make
```

Comment:
I wanted to show the case where you have a local branch but yeah, HEAD is ugly. I guess something like `local-branch` would work or I'll just use master and avoid problems :)